src/get-started/java-binding.md

```diff
@@ -180,11 +198,23 @@ current storage state when the transaction is executed. Exonum passes `Fork`
 as an argument – a view that allows performing modifying operations. A service
 schema object can be used to access data collections of this service.
 
+Also, `Transaction#execute` method may throw `TransactionExecutionException`
+which contains a transaction error report. This feature allows users to notify
+Exonum about an error in a transaction execution whenever one occurs.
+The service checks the preconditions before executing a transaction and either
+accepts it or issues an exception that is further transformed into an Exonum
+core [TransactionResult enum][transaction-result] containing an error code and
+a message with error data.
+If transaction execution fails, the changes invoked by the transaction are
```

Comment:
I think we do not "invoke" changes to the implied object (a database, a storage), but "make" them, WDYT?